### PR TITLE
Updated lumina-fm man page

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

**Comment by a-stjohn** [comment]
I noticed I didn't add "Ar" before "FILE" in the synopsis. Also, I forgot to put ".El"  for the list in the section "DESCRIPTION" and also missed an ".El" at the end in Section "OPTIONAL RUNTIME DEPENDENCIES"

File: manpages/lumina-fm.8 [hunk 1]
```diff
@@ -1,37 +1,60 @@
...
-.Nd is a utility used for browsing or otherwise interacting with files on the system.
+.Nd is a utility used for browsing and interacting with files
+on the system.
 
 .Sh SYNOPSIS
 .Nm
-.Op <argument>
-.Op <file path>
+.Op Fl new-instance
+.Op FILE
```
Comment (Mrt134) [inline manpages/lumina-fm.8]: Needs Ar before FILE -> .Op Ar FILE

File: manpages/lumina-fm.8 [hunk 2]
```diff
@@ -1,37 +1,60 @@
...
-maintain the files stored on the system.
+is the CLI command to open Insight File Manager.
+The file manager interface maintains files stored on the system.
 .Nm
-can take an optional argument and file path. If no argument or file path is
-specified, the command will open the Insight File Manager in the home
-directory. Specifying a file path while the interface is open creates a new
-tab in the existing window.
+can take an optional argument and file path.
+If no argument or file path is specified, the command will open the
```
Comment (Mrt134) [inline manpages/lumina-fm.8]: ...the command will open... -> the command opens

File: manpages/lumina-fm.8 [hunk 3]
```diff
@@ -1,37 +1,60 @@
...
+Insight File Manager in the home directory.
+Specifying a file path while the interface is open creates a new tab in
+the existing window.
 .Pp
 .Bl -tag -width indent
 .It Ic -new-instance
 Opens Insight File Manager in a new window.
```
Comment (Mrt134) [inline manpages/lumina-fm.8]: Needs an .El entry to close list.

File: manpages/lumina-fm.8 [hunk 4]
```diff
@@ -1,37 +1,60 @@
...
+.Pp
+.Bl -tag -width indent
+.It Ic lumina-archiver
+Gives the option to instantaneously extract an
+archive into a new subfolder within the current directory.
+.It Ic lumina-open
+Gives the option to "open-with".
+.It Ic lumina-fileinfo
+Provides an option to view the properties of files.
+
```
Comment (Mrt134) [inline manpages/lumina-fm.8]: Needs .El entry to close list.

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Added notes for the changes you mentioned were missing, plus a couple minor text reworks.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**